src/index.js

```diff
@@ -164,25 +170,14 @@ class ZyteSmartProxyPuppeteer {
 
     async launch(options) {
         await this._configure_zyte_smartproxy_puppeteer(options)
-        let args = [
```

Comment:
Hi @realslimshanky 
Nice to hear you! :)
Do you remember why do we need these arguments?

Reply:
I remember the support article was suggesting these settings for puppeteer.